### DISCUSSION TITLE
mmex_1.5.20-Ubuntu.22.04.jammy_amd64.deb std:: bad_alloc

### DISCUSSION THREAD

**Discussion by pivantable** [moneymanagerex/moneymanagerex#5038] [Q&A]
impossible d'ouvrir mmex depuis la version 1.5.19
merci pour l'aide
![Capture d’écran du 2022-09-04 11-51-01](https://user-images.githubusercontent.com/108255685/188308013-3c8f4493-1c59-4b34-b79f-77ab088a6ed5.png)

**Comment by PMaff** [comment]
@pivantable  can you do a
`lsb_release -a`
in a terminal window?
Please copy the output here.

**Reply by pivantable** [reply]
![Capture d’écran du 2022-09-10 18-08-08](https://user-images.githubusercontent.com/108255685/189491943-8534b4d4-e41e-4678-bf60-1d6d019a25d5.png)

**Comment by PMaff** [comment]
Subject says you tried 1.5.20 but above you are talking about 1.5.19 (I do not speak French).

Did you try
[Linux Ubuntu 22.04 Jammy (64-bit)](https://github.com/moneymanagerex/moneymanagerex/releases/download/v1.5.20/mmex_1.5.20-Ubuntu.22.04.jammy_amd64.deb)
from here:
https://moneymanagerex.org/docs/downloads/latest/
?

**Comment by PMaff** [comment]
We are also trying a few things here in this thread in the forum:
https://forum.moneymanagerex.org/viewtopic.php?p=23324#p23324
